pyproject.toml

```diff
@@ -27,19 +27,19 @@ classifiers=[
     "Programming Language :: Python :: 3.12"
 ]
 dependencies = [
-"numpy>=1.26.4,<2.0",
-"accelerate>=0.20.3,!=0.34,<1.7",
-"transformers>=4.53.0,<=4.55.4",
-"torch>2.6.0,<=2.8.0",
+"numpy>=1.26.4,<2.2.0",
+"accelerate>=1.9.0,<2.0.0",
+"transformers>=4.55.0,<=4.55.4",
+"torch>2.7.0,<2.9.0",
 "sentencepiece>=0.1.99,<0.3",
-"tokenizers>=0.13.3,<1.0",
+"tokenizers<=0.22",
 "tqdm>=4.66.2,<5.0",
-"trl>=0.13,<0.20",
-"peft>=0.15.0,<=0.15.2",
-"protobuf>=5.28.0,<6.0.0",
-"datasets>=3.5.0,<4.0",
+"trl>=0.19.1,<0.20.0",
+"peft<=0.17.0",
```

Comment:
don't we want the `>=0.15` filter for peft?

Reply:
sure we can add...

Reply:
we need peft higher version will set accordingly